src/client/src/models/Task.js

```diff
@@ -1,43 +1,43 @@
 import {TimeLog} from './TimeLog';
 
 export default class Task {
-    constructor(id, title, description, colour) {
-        this.id = id;
-        this.title = title;
-        this.description = description;
-        this.colour = colour;
-        this.status = status;
-        this.employees = [];
-        this.assigned = false;
-        this.inProgress = false;
-        this.finished = false;
-        this.timeLogs = [];
-    }
+	constructor(id, title, description, colour) {
+		this.id = id;
+		this.title = title;
+		this.description = description;
+		this.colour = colour;
+		this.status = status;
```

Comment:
Where does the status parameter come from?

Reply:
True, I have not yet updated the task model from when we originally made it. Will do this.